lib/paperclip/thumbnail.rb

```diff
@@ -63,8 +63,7 @@ def convert_options?
     # that contains the new image.
     def make
       src = @file
-      dst = Tempfile.new([@basename, @format ? ".#{@format}" : ''])
-      dst.binmode
+      dst = TempfileFactory.new.generate([@basename, @format ? ".#{@format}" : ''].join)
```

Comment:
Line is too long. [88/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.